### PR TITLE
[sc-324] Improve syntax for tagged terms

### PR DESCRIPTION
**Pull request by imaqtkatt**

### CODE REVIEW

**Review by developedby — CHANGES_REQUESTED** [review]
Can you also do this for `dup`? For consistency

**Review by LunaAmora — CHANGES_REQUESTED** [review]
Just need to update the readme with the tag changes

**Review by LunaAmora — APPROVED** [review]

**Review by developedby — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -264,19 +264,19 @@ Similarly to dups and sups, lambdas and applications can have labels too.
...
 // data Bool = T | F
-T = λ#Bool t λ#Bool f t
-F = λ#Bool t λ#Bool f f
+T = #Bool @t #Bool @f t
+F = #Bool @t #Bool @f f
 
 // data List = (Cons x xs) | Nil
-Cons = λx λxs λ#List c λ#List n (#List.Cons.xs (#List.Cons.x c x) xs)
-Nil  =        λ#List c λ#List n n
+Cons = @x @xs #List @c #List @n #List (c x xs)
```
Comment (LunaAmora) [inline README.md:271]: Here the application should be `#List.Cons.xs (#List.Cons.x (c x) xs)`

**Review by LunaAmora — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LunaAmora) [reply README.md:271]: also, ins'nt a little better to use `λ` to mantain consistence on the examples?

Comment (imaqtkatt) [reply README.md:271]: It is correct, this is showing how to encode data types with tagged lambdas, not what a `data` definition is desugared into

**Review by LunaAmora — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (imaqtkatt) [reply README.md]: Fixed, I had not noticed this correctly before ;)

**Pull request merged**